concent_api/core/views.py

```diff
@@ -505,6 +521,40 @@ def handle_receive_delivered_force_report_computed_task(request, delivered_messa
     return ack_report_computed_task
 
 
+def handle_receive_force_subtask_results(
+    request,
+    decoded_message:                message.concents.ForceSubtaskResults,
+    previous_message_from_database: Union[ReceiveStatus, Message],
+    message_model
+) -> message.concents.ForceSubtaskResults:
+
+    assert decoded_message.TYPE in message.registered_message_types
```

Comment:
You can be more specific here:

``` python
assert decoded_message.TYPE == message.concents.ForceSubtaskResults.TYPE
```

Your could also use `isinstance()` instead.

Reply:
I've change it to `isinstance()`.